site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1041,10 +1041,14 @@ public function saveSingleComponent() {
             $component->deleteData($gradeable->getGdId());
             $debug = 'delete';
         } else {
-            //only change the component information is the mark was modified or componet and its gradeable are out of sync.
-            if ($component->getGrader() === null || $overwrite === "true") {
-                $component->setGrader($this->core->getUser());
-            }
+                //only change the component information is the mark was modified or componet and its gradeable are out of sync.
```

Comment:
'... component information *if* the mark was modified or *component* and ...' I know they aren't your spelling errors, but since you're modifying the line anyway can you fix them?